### ISSUE TITLE
thanks should concentrate on funding the project, not the author

### ISSUE THREAD

**Issue by tomdottom** [phildini/thanks#20]
Whilst funding links may belong to an author, they might not.

Primary focus should be on funding and giving thanks to on a project basis
whether that be to an authors, group, or organisation.

## Proposed output
```
Scanning your requirements file...
Loading data about projects...
You depend on x projects who would enjoy donations!

  Project     Where to thank                    Author

  click       https://www.paypal.me/untitaker   untitaker
  thanks      https://patreon.com/phildini      phildini, tomdottom
  virtualenv  https://earn.com/brosner/         brosner, jezdez
```

## Current output
```bash
$ thanks foo.txt

Loading data about contributors...
Scanning your requirements file...
You depend on 4 authors who would enjoy donations!

  Author      Where to thank                    Packages

  untitaker   https://www.paypal.me/untitaker   click
  phildini    https://patreon.com/phildini      thanks
  brosner     https://earn.com/brosner/         virtualenv
  jezdez      https://www.paypal.me/jezdez      virtualenv
```

**Comment by phildini** [comment]
I think this is the right direction, but I want the ability to pass in `--include-authors` or somesuch so that if we _have_ author funding data we can provide it.

**Comment by tomdottom** [comment]
Addressed by #21
